osu.Game/Screens/Select/FooterButtonMods.cs

```diff
@@ -90,9 +90,9 @@ protected override void LoadComplete()
 
             MultiplierText.Text = multiplier.Equals(1.0) ? string.Empty : $"{multiplier:N2}x";
 
-            if (multiplier > 1.0)
+            if (multiplier >= 1.005)
                 MultiplierText.FadeColour(highMultiplierColour, 200);
-            else if (multiplier < 1.0)
+            else if (multiplier < 0.995)
```

Comment:
should use [`Precision.DefinitelyBigger()`](https://github.com/ppy/osu-framework/blob/a86dfaa17a203c4d13a89ea30a10e423d11ff466/osu.Framework/Utils/Precision.cs#L34-L41) instead

Reply:
Thank you for your review.
Fixed.